pom.xml

```diff
@@ -85,12 +85,12 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-cps</artifactId>
-        <version>2.61.1</version>
+        <version>2.67</version>
```

Comment:
Preparing for test of https://github.com/jenkinsci/workflow-cps-plugin/pull/211.